source/code/plugins/agent_maintenance_script.rb

```diff
@@ -317,18 +335,19 @@ def form_post_request_and_http(headers, uri_string, body, cert, key)
     def heartbeat
       # Reload config in case of updates since last heartbeat
       @load_config_success = load_config
```

Comment:
Since this is no longer semantically a Boolean, it might make sense to rename it to load_config_result_code, or something similar.